components/lvgl/Kconfig

```
@@ -712,7 +772,23 @@ menu "LVGL configuration"
             default n
     endmenu
 
-    menu "Widgets usage"
+    menu "Widgets"
+        config LVGL_USE_OBJ_REALIGN
+            bool "Enable `lv_obj_realaign()` based on `lv_obj_align()` parameters."
```

We may as well correct the spelling here at the same time.

```suggestion
            bool "Enable `lv_obj_realign()` based on `lv_obj_align()` parameters."
```

---

I will, this text is taken from the `lv_config_template.h`, we may fix it over there aswell.

---

It seems already fixed in [lv_conf_templalte.h](https://github.com/lvgl/lvgl/blob/master/lv_conf_template.h#L538).